---
title: use-env-prefix
sidebarDepth: 0
description: Use prefix `GRIDSOME_` when using process.env in browser
---

## Rule's description is here (gridsome/use-env-prefix)

## :book: Rule Details

:-1: Examples of **incorrect** code for this rule:

```html
Bad example
```

:+1: Examples of **correct** code for this rule:

```html
Good example
```

## :wrench: Options

If this rule has option, write here.
